src/components/v2/devtronStackManager/DevtronStackManager.component.tsx

```diff
@@ -24,8 +24,10 @@ import { ReactComponent as File } from '../../../assets/icons/ic-file-text.svg'
 import { ReactComponent as Chat } from '../../../assets/icons/ic-chat-circle-dots.svg'
 import { ReactComponent as Info } from '../../../assets/icons/info-filled.svg'
 import { ReactComponent as Warning } from '../../../assets/icons/ic-warning.svg'
+import { ReactComponent as Note } from '../../../assets/icons/ic-note.svg'
+import { ReactComponent as CloseIcon } from '../../../assets/icons/ic-close.svg'
 
```

Comment:
please remove the empty line (29) if not done purposely

Reply:
done